src/lib/UI/Module/ContentTree/NodeFactory.php

```diff
@@ -95,7 +95,25 @@ public function createNode(
     ): Node {
         $uninitializedContentInfoList = [];
         $containerLocations = [];
-        $node = $this->buildNode($location, $uninitializedContentInfoList, $containerLocations, $loadSubtreeRequestNode, $loadChildren, $depth, $sortClause, $sortOrder);
+
+        $userBookmarks = $this->bookmarkService->loadBookmarks(
+            0,
+            $this->getSetting('children_load_max_limit')
+        );
+        $bookmarkLocations = array_flip(array_column($userBookmarks->items, 'id'));
```

Comment:
This logic is valid only if user have less then `$this->getSetting('children_load_max_limit')` bookmarks.

Reply:
@adamwojs fixed in https://github.com/ibexa/admin-ui/pull/744/commits/37ca548b9798f20c55df6353590744536101ebca